llvm/test/CodeGen/LoongArch/target-abi-from-triple-edge-cases.ll

```diff
@@ -1,3 +1,4 @@
+; NOTE: Assertions have been autogenerated by utils/update_llc_test_checks.py UTC_ARGS: --version 5
```

Comment:
The checking prefixes for this test were written manually. I prefer keeping this style unchanged this time for easy-reviewing. Then maybe you can submit a separate `NFC` PR switching to use `update_llc_test_checks.py`.

Reply:
Done.